components/quiz/Quiz.tsx

```diff
@@ -0,0 +1,33 @@
+import { QuizProps } from "../../utils/type";
```

Comment:
The PropTypes should be declared in the same file

Reply:
> The PropTypes should be declared in the same file

don't know what you exactly mean?!